apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -419,6 +425,7 @@ const Ticket = new GQLListSchema('Ticket', {
                 const classifierResult = await classifyTicket(context, resolvedData.details)
 
                 resolvedData.classifier = get(classifierResult, 'id')
+                resolvedData.isAutomaticallyClassified = true
```

Comment:
Why this flag set's to true on every request? Shouldn't it depend on the result above?

Reply:
Tickets created or updated from CRM are considered to be classified manually, because the operator has validated this classifier.
Fixed: now `isAutoClassified: true` is passed only if this ticket has `isAutoClassified: false`.